docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+  - [Prerequisites](#prerequisites)
+  - [Registering for the SGX Attestation Service Utilizing EPID](#registering-for-the-sgx-attestation-service-utilizing-epid)
+    - [Known Issues](#known-issues)
+  - [Building the FPC dev container](#building-the-fpc-dev-container)
+  - [Final preparation of the dev-container](#final-preparation-of-the-dev-container)
+
+## Creating the Confidential Computing Instance on Azure.
+Use the [Quick Create Portal](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal) to create
+the virtual machine.
+Use the following parameters:
+* OS = *Linux (ubuntu 20.04)*
+* Size = Standard DC2s v2 (2 vcpus, 8 GiB memory)
+
+## Prerequisites
+Connect to your instance using ssh and execute the following commands
+```bash
+#update machine
+sudo -i
+apt-get update
+apt-get upgrade
+apt install -y docker docker-compose golang make
+exit
+#install sgx environment
+#1.add sgx repo + key and then install packages form this repository
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+sudo apt-get update
+sudo apt -y install libssl-dev libsgx-enclave-common libsgx-enclave-common-dev libsgx-ae-qe3 libsgx-ae-qve libsgx-epid libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex libsgx-uae-service libsgx-urts
+sudo reboot
+```
+After reboot is completed connect to your vm again.
+```bash
+#check if asemd service is running
+sudo systemctl status aesmd.service
+```
+```bash
+sudo usermod -aG docker $(whoami)
+go env -w GO111MODULE=auto
+
+echo 'export GOPATH=$HOME/go' >> .bashrc
+echo 'export GOBIN=$GOPATH/bin' >> .bashrc
+echo 'export PATH=$PATH:$GOBIN' >> .bashrc
+
+sudo reboot
+``` 
+After reboot is completed connect to your vm again.
+```bash
+mkdir -p $GOPATH
+cd $GOPATH
+go get github.com/hyperledger/fabric-private-chaincode
+cd  ~
+echo 'export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode' >> .bashrc
+#you can also re-log than you don't need to set it
+export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode 
+```
+
+## Registering for the SGX Attestation Service Utilizing EPID
+* Get a account [here](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html)
+* Sign in.
+* Once you are signed in go [here](here: https://api.portal.trustedservices.intel.com/EPID-attestation)
+* Subscribe for development unlikable.
+* You will receive a SPID, Primary Key and Secondary Key.
+* Use these information to replace the appropriate places in the commands below.
+
+```bash
+echo '[YOUR_SPID]' > ${FPC_PATH}/config/ias/spid.txt
+echo '[YOUR_PRIMARY_KEY]' > ${FPC_PATH}/config/ias/api_key.txt
+echo 'epid-unlinkable' > ${FPC_PATH}/config/ias/spid_type.txt
+```
+
+### Known Issues
```

Comment:
It's not immediately clear to me what are the known issues here? Also re-formatting may help better read it, for instance, one paragraph per issue or some form of list?

Reply:
The issue is not with FPC, but rather with the intel subscribe process which runs into a broken link if you want to create an account. You need to visit a different website in order to register and then you are able to subscribe.

I removed this section, because its intel specific and not related to FPC